chaos_genius/connectors/base_db.py

```diff
@@ -4,9 +4,24 @@
 from sqlalchemy import exc as sqlalchemy_exc
 from sqlalchemy import text
 
+
 logger = logging.getLogger(__name__)
 
+
 class BaseDb:
+
+    SQL_IDENTIFIER = ""
```

Comment:
We should implement the getter function for these constants since these can vary with the data source. Those respective getter functions should be part of every DB connector class.
While doing this, let's also make sure that the data inside the Chaos Genius is consistent and in the same format irrespective of the data source variation.

Reply:
@manassolanki I don't think a getter is needed here since it's just a constant. This class constant will be a part of every class and any subclass can set its own if it needs to override. Example: https://github.com/chaos-genius/chaos_genius/blob/2d8245d317dee8732d14af6a25419aab35f3c2cf/chaos_genius/connectors/postgres.py#L8-L10

Reply:
This is a read-only attribute of the class. So the better pattern for these use cases is declaring the variable using the __VARIBLE_NAME and then using the getter method. The reason is that any class instance won't be able to modify the constant values. (Assuming the developer knows the python convention and doesn't try to mess up with the private attributes and methods)

Reply:
That makes sense. I will get on it.

Reply:
Done